tracing/src/lib.rs

```diff
@@ -146,6 +146,8 @@
 //!
 //! For example:
 //! ```
+//! # #[cfg(feature = "attributes")]
```

Comment:
I don't believe this works...my understanding is that feature flags are not passed to doctests, so this will silently never be compiled. If that's the case, I have a slight preference for just making it `ignore`d.

Reply:
alternatively, we could probably use `cfg_attr` and replace the doctest with
```rust
#[cfg_attr(feature = "attributes", doc = "```\nuse tracing::{Level, event, instrument};\n\n....")]
```
but that is kind of awful.

Reply:
It does seem to have worked?